main/src/modules/reactiveLayer/behaviorManager/src/pointing.cpp

```diff
@@ -11,10 +11,10 @@ void Pointing::run(Bottle args/*=Bottle()*/) {
     yInfo() << "Pointing::run";
     Bottle *sensation = sensation_port_in.read();
     
-    int id = yarp::os::Random::uniform(0, sensation->size() - 1);
-    yDebug() << "Randomly selected: " << id;
+    int id = yarp::os::Random::uniform(0, sensation->size()-1);
+    yDebug()<<"Randomly selected: "<< id; // should be random here 
     string obj_name = sensation->get(id).asList()->get(1).asString();
-
+    
```

Comment:
Hi, are there meant to be any changes here? It seems like this is taken from an old version.